packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -25,10 +27,17 @@ const TradersHubOnboarding = observer(() => {
                         data_testid='dt_traders_hub_onboarding_icon'
                         icon={is_dark_mode_on ? 'IcAppstoreTradingHubOnboardingDark' : 'IcAppstoreTradingHubOnboarding'}
                         size={20}
-                        onClick={() => {
-                            history.push(routes.onboarding);
-                            setIsOnboardingVisited(false);
-                        }}
+                        onClick={
```

Comment:
minor - for complex funcitons, I tend to put them to separate functions instead of inlining

Reply:
@wojciech-deriv ok, I'll add this change to this PR (https://github.com/binary-com/deriv-app/pull/10807), because it extends this PR.